ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -74,16 +83,21 @@ sub handle_config_file
             my $ref = ref($val);
             if($ref) {
                 if($ref eq 'ARRAY') {
+                    if ($option_set eq 'multiline_options') {
+                        $fh->remove_lines(qr{(?i)^\W*$option(?:\s+\S+)+}, qr{^#});
```

Comment:
this will always remove content that you add below. why does it it need to be done this way? add a small comment why this is done